src/drive/web/modules/drive/RenameInputV2.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { connect } from 'react-redux'
+
+import { withClient } from 'cozy-client'
+import Alerter from 'cozy-ui/transpiled/react/Alerter'
+
+import FilenameInput from 'drive/web/modules/filelist/FilenameInput'
+import { abortRenaming } from './renameV2'
+
+const updateFileNameQuery = async (client, file, newName) => {
+  return client
+    .collection('io.cozy.files')
+    .updateAttributes(file.id, { name: newName })
+}
+
+const RenameInput = ({ abortRenaming, file, client }) => {
+  return (
+    <FilenameInput
+      name={file.name}
+      onSubmit={async newName => {
+        try {
+          await updateFileNameQuery(client, file, newName)
+        } catch {
+          Alerter.error('alert.folder_name', { folderName: name })
```

Comment:
Is it expected to have "folderName" here (instead of fileName) ?

Reply:
I used the existing string. I created a new one. And thanks for the comment, it appears that I used the old `name` instead of `newName`...